File: CodeChef/Self Learning/Getting Started/Lucky_Four.py
# CodeChef - LUCKFOUR - Lucky Four
# https://www.codechef.com/LP0TO101/problems/LUCKFOUR

import sys
input = sys.stdin.readline

T = int(input())

for _ in range(T):
    N = input().rstrip()
    ans = N.count('4')
    print(ans)

File: CodeChef/Self Learning/Getting Started/Lucky_Four.cpp
// CodeChef - LUCKFOUR - Lucky Four
// https://www.codechef.com/LP0TO101/problems/LUCKFOUR

#include <bits/stdc++.h>
using namespace std;

int main(){
    ios::sync_with_stdio(0);
    cin.tie(0);

    int T;
    cin >> T;

    for(int i = 1; i <= T; ++i){
        string N;
        cin >> N;

        int ans = count(N.begin(), N.end(), '4');
        cout << ans << "\n";
    }


}